packages/component-submission/server/resolvers/submitManuscript.js

```diff
@@ -58,39 +93,13 @@ async function submitManuscript(_, { data }, { user, ip }) {
   )
 
   // This function can take a while so do not await this (apart from in tests)
-  return mecaExport(manuscriptModel, S3Storage.getContent, ip)
-    .then(async () => {
-      logger.info(`Manuscript ${manuscriptModel.id} successfully exported`)
-      const manuscript = new Manuscript(config, userUuid, S3Storage)
-      const notify = new Notification(config)
-      await notify.sendFinalSubmissionEmail(manuscriptModel)
-      await ManuscriptModel.updateStatus(
-        manuscriptModel.id,
-        ManuscriptModel.statuses.MECA_EXPORT_SUCCEEDED,
-      )
+  _runExport(manuscriptModel, userUuid, ip)
 
-      return manuscript.getView(manuscriptModel.id)
-    })
-    .catch(async err => {
-      logger.error('MECA export failed', err)
-      await ManuscriptModel.updateStatus(
-        manuscriptModel.id,
-        ManuscriptModel.statuses.MECA_EXPORT_FAILED,
-      )
-      return mailer.send({
-        to: config.get('meca.email.recipient'),
-        from: config.get('meca.email.sender'),
-        subject: `${config.get('meca.email.subjectPrefix')}MECA export failed`,
-        text: `Manuscript ID: ${manuscriptModel.id}
-Manuscript title: ${manuscriptModel.meta.title}
-Error:
+  return (async () => undefined)().then(async () => {
```

Comment:
Why does this need to be a promise? the resolver can return `manuscript.getView`  and the request response will still be the resolved value without awaiting it unless I'm mistaken?